docs/modules/ROOT/pages/mathematical-operators.adoc

```diff
@@ -0,0 +1,93 @@
+[[mathematical-operators]]
+= Mathematical operators
+
+Mathematical operators are a handy way to update numerical fields in a single roundtrip to the DB.
```

Comment:
I'm not sure if `roundtrip` is the best word here. Also, this refers not to any updates but to updates based on their values. Maybe:

```suggestion
Mathematical operators are a handy way to update numerical fields based on their original values in a single DB transaction.
```